app/helpers/fhir_helper.rb

```diff
@@ -37,6 +37,43 @@ def patient_as_fhir(patient)
       creator_agent_ref.display = patient.creator.email
     end
 
+    contact_info = {
+      contact: []
+    }
+    if patient.contact_type == 'Self'
+      contact_info[:telecom] = [
+        to_contact_point(patient.primary_telephone, 'phone', 1, patient.primary_telephone_type, false),
+        to_contact_point(patient.secondary_telephone, 'phone', 2, patient.secondary_telephone_type, false),
+        to_contact_point(patient.email, 'email', 1, nil, false),
+        to_contact_point(patient.international_telephone, 'phone', 1, nil, true)
+      ].reject(&:nil?)
+    else
+      contact_info[:contact] << {
+        telecom: [
+          to_contact_point(patient.primary_telephone, 'phone', 1, patient.primary_telephone_type, false),
+          to_contact_point(patient.secondary_telephone, 'phone', 2, patient.secondary_telephone_type, false),
+          to_contact_point(patient.email, 'email', 1, nil, false),
+          to_contact_point(patient.international_telephone, 'phone', 1, nil, true)
+        ].reject(&:nil?),
+        relationship: [FHIR::CodeableConcept.new(coding: [FHIR::Coding.new(code: patient.contact_type)])],
+        name: FHIR::HumanName.new(text: patient.contact_name)
+      }
+    end
+    if %i[alternate_contact_name alternate_contact_type alternate_primary_telephone alternate_secondary_telephone alternate_email
+          alternate_international_telephone].any?(&:present?)
+      contact_info[:contact] << {
+        telecom: [
+          to_contact_point(patient.alternate_primary_telephone, 'phone', 1, patient.alternate_primary_telephone_type, false),
+          to_contact_point(patient.alternate_secondary_telephone, 'phone', 2, patient.alternate_secondary_telephone_type, false),
+          to_contact_point(patient.alternate_email, 'email', 1, nil, false),
+          to_contact_point(patient.alternate_international_telephone, 'phone', 1, nil, true)
+        ].reject(&:nil?),
+        relationship: [FHIR::CodeableConcept.new(coding: [FHIR::Coding.new(code: patient.alternate_contact_type)])],
```

Comment:
Sticking point for me with this- I don't like the possibility of having "Self" as the `contact.relationship`, since `Patient.telecom` should really always hold that value, even if Self is the alternate contact in Sara Alert. But in that case we'd then need a way to indicate when the non-Self `contact` is the _primary_ contact. I can't think of any other way to address this than to use an extension.

The `alternate-contact` extension already exists here, but my preference would be to modify that to instead be `primary-contact`, indicating that it should be used over `Patient.telecom`.